integration/Makefile

```diff
@@ -61,6 +61,10 @@ verification-tests:
 network-tests:
 	go test $(if $(VERBOSE),-v,) $(if $(JSON_OUTPUT),-json,) $(if $(NUM_RUNS),-count $(NUM_RUNS),) -tags relic ./tests/network/...
 
+.PHONY: observer-tests
+observer-tests:
+	GO111MODULE=on go test $(if $(VERBOSE),-v,) $(if $(JSON_OUTPUT),-json,) $(if $(NUM_RUNS),-count $(NUM_RUNS),) -tags relic ./tests/observer/...
```

Comment:
`GO111MODULE=on` is no longer needed on the make targets. Please remove for code consistency.

Reply:
Thank you for the review @AndrewM-SDET. Fixed: [befe665](https://github.com/GetElastech/flow-go/commit/befe6658e16bfe0e0c2e02677b9bd1904072f12f)